src/concourse/pipelines/open_edx/mfe/values.py

```diff
@@ -97,6 +97,7 @@
         site_name="MITx Online QA",
         studio_domain="studio-qa.mitxonline.mit.edu",
         support_url="mitxonline.zendesk.com/hc/",
+        schedule_email_section="true",  # Because the communication MFE treats this boolean as string  # noqa: E501
```

Comment:
It's probably worth copying this to the production block as well (line 120)

Reply:
I intentionally kept its scope to QA so that we can test this first and then create a follow-up PR and add this to Prod. But let's hear @pdpinch 's thoughts on this. 

@pdpinch Should we enable it for both QA and Prod?